docker-compose.yml

```diff
@@ -8,34 +8,33 @@ services:
     extends:
       service: base
       file: docker-compose-base.yml
-    image: registry.scc.suse.de/postgresql:9.4
+    image: postgres
     volumes:
-      - db_storage:/var/lib/pgsql/data
+      - db_storage:/var/lib/postgresql/data
+    environment:
+      - PGDATA=/var/lib/postgresql/data/pgdata
 
   rmt:
     extends:
       service: base
       file: docker-compose-base.yml
     build: .
     environment:
-      - RAILS_ENV=test
       - POSTGRES_HOST=db
-    volumes:
-      - ./:/srv/www/rmt
+      - RAILS_ENV=production
+      - SECRET_KEY_BASE=8ea53ad3bc6c03923e376c8bdd85059c1885524947a7efe53d5e9c9d4e39861106ffd6a2ece82b803072ed701e6c960bade91644979e679416c5f255007237ae
```

Comment:
that's a copy of development key from `secrets.yml` already committed here. The workflow for production servers is different anyway: either with env variable or with new encrypted key from Rails 5.1